tests/ChunkedEncoding.hx

```diff
@@ -0,0 +1,34 @@
+package ;
+
+using tink.CoreApi;
+import tink.http.Fetch;
+
+
+@:asserts
+@:timeout(10000) 
+class ChunkedEncoding{
+  public function new(){}
+  public function make_request(){
+    var assertion = Future.trigger();
+    var res = Fetch.fetch(
```

Comment:
I think unit tests shouldn't involve the web.

Can you instead add a test to TestChunked with inline `Bytes`?

Reply:
or you can move the web test to the TestHttp class. because we have some conditionals for running actual net-involved tests, separated from pure unit tests...

Reply:
It'll probably be next week, but to be fair if the conditions a) travis doesn't have an internet connection and b) github is offline are met, we've got bigger problems.:)

Reply:
I've done both. I still haven't been able to run `master.hxml` locally, so still have `tests_0b1kn00b`